lib/active_merchant/billing/gateways/quickpay/quickpay_v10.rb

```diff
@@ -214,7 +214,9 @@ def invalid_operation_code(response)
         end
 
         def invalid_operation_message(response)
-          response['operations'].last['qp_status_msg']
+          if response['operations']
```

Comment:
This could be `response['operations'] && response['operations'].last['qp_status_msg']`, right?